tool/tctl/common/helpers_test.go

```diff
@@ -0,0 +1,115 @@
+package common
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"io"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/gravitational/teleport/lib/config"
+	"github.com/gravitational/teleport/lib/service"
+	"github.com/gravitational/teleport/lib/utils"
+
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v2"
+)
+
+func runResourceCommand(t *testing.T, fc *config.FileConfig, args []string, ) (*bytes.Buffer, error) {
+	var stdoutBuff bytes.Buffer
+	command := &ResourceCommand{
+		stdout: &stdoutBuff,
+	}
+	cfg := service.MakeDefaultConfig()
+
+	app := utils.InitCLIParser("tctl", GlobalHelpString)
+	command.Initialize(app, cfg)
+
+	selectedCmd, err := app.Parse(args)
+	require.NoError(t, err)
+
+	var ccf GlobalCLIFlags
+	ccf.ConfigString = mustGetBase64EncFileConfig(t, fc)
+
+	clientConfig, err := applyConfig(&ccf, cfg)
+	require.NoError(t, err)
+
+	client, err := connectToAuthService(context.Background(), cfg, clientConfig)
+	require.NoError(t, err)
+
+	_, err = command.TryRun(selectedCmd, client)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Should this be `require.NoError(t, err)`, or is this returning error on purpose?

Reply:
The return error is checked by the caller - in one case the returned error is NotFound